src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -1916,5 +1916,65 @@ private Task<string> ResetConsole()
         {
             return ConsoleCmdAsync(@"shell echo -e \\033c 1>&2");
         }
+
+        public bool MapCurrentSrcToCompilerSrc(string currentSrc, out string compilerSrc)
+        {
+            StringComparison comp = PlatformUtilities.IsWindows() ? StringComparison.OrdinalIgnoreCase : StringComparison.Ordinal;
+            foreach (var e in _launchOptions.SourceMap)
+            {
+                if (e.UseForBreakpoints && currentSrc.StartsWith(e.EditorPath, comp))
+                {
```

Comment:
Should you replace the directory separators for both so that they match? I know we update some of the directory separators in some instances but I don't remember if we do it for breakpoints.